src/biome/text/_model.py

```diff
@@ -312,74 +319,70 @@ def predict_batch(
             raise WrongValueError(
                 f"Failed to make predictions for '{input_dicts}'"
             ) from error
+
         for input_dict, prediction in zip(input_dicts, predictions):
             self.log_prediction(input_dict, prediction)
```

Comment:
Just a comment. With this refactor all predictions (from predict and explain actions) will be logged.

It should be nice only apply prediction logging to predict actions. (But it's not a PR requirement at all)

Reply:
Yes, i was not sure which behavior we prefer. i will change it, it's an easy fix.